docker/models/containers.py

```diff
@@ -0,0 +1,144 @@
+from docker.errors import ImageNotFound
+from ..errors import ContainerError
+from .images import Image
+from .resource import Collection, Model
+
+
+class Container(Model):
+
+    @property
+    def name(self):
+        """
+        Returns the name of the container.
+        """
+        if self.attrs.get('Name') is not None:
+            return self.attrs['Name'].lstrip('/')
+
+    @property
+    def status(self):
+        """
+        Returns the status of the container. For example, `running`, or
+        `exited`.
+        """
+        return self.attrs['State']['Status']
+
+    def attach(self, *args, **kwargs):
```

Comment:
Hopefully the docs will be good enough that they don't have to look at the source code. ;)

Agreed though, it's a bit annoying. If this turns out to be a nuisance down the line, we can trivially duplicate the args here.